yamcs-client/examples/file_transfer.py

```diff
@@ -37,3 +38,40 @@
         print("Upload failure:", upload.error)
     else:
         print(f"Successfully uploaded {upload.remote_path} ({upload.size} bytes)")
+
```

Comment:
Can we put some of the examples on the web page under snippets? Cfr how it's done for others.
https://docs.yamcs.org/python-yamcs-client/filetransfer/